File: frontend/src/app/models/release-notes.ts
export class ReleaseNotes {
    noteId: number;
    slug: string;
    date: string;
    body: string;

    constructor (noteId: number, slug: string, date: string, body: string) {
        this.noteId = noteId;
        this.slug = slug;
        this.date = date;
        this.body = body;
    }
}

File: server/src/main/java/dev/jaczerob/eras/server/models/ttr/releasenotes/ReleaseNotes.java
package dev.jaczerob.eras.server.models.ttr.releasenotes;

public class ReleaseNotes extends ReleaseNotesPartial {
    private String body;

    public ReleaseNotes() {
        super();
        this.body = "unknown";
    }

    public ReleaseNotes(final int noteId, final String slug, final String date, final String body) {
        super(noteId, slug, date);
        this.body = body;
    }

    public String getBody() {
        return this.body;
    }

    public void setBody(final String body) {
        this.body = body;
    }
}
